v19.2/performance-tuning-insecure.md

```diff
@@ -511,10 +510,6 @@ $ cockroach sql \
 --execute="CREATE INDEX ON rides (start_time) STORING (rider_id);"
 ~~~
 
-{{site.data.alerts.callout_info}}
-The `rides` table contains 1 million rows, so adding this index will take a few minutes.
-{{site.data.alerts.end}}
```

Comment:
Is this no longer true?

Reply:
It takes like 5 seconds, so I don't think we should have a note about it taking "a few minutes".